### PR TITLE
Deneb: Add test cases to execution payload processing

### PR DESCRIPTION
**Pull request by marioevz**
Adds a couple of simple test cases I thought could be interesting.

I added a small comment of my understanding of the goal of the tests in this file, and based on that is what I thought was missing and added, but if this was a misinterpretation please let me know.

I'll try to think of more tests to add.

### CODE REVIEW

File: tests/core/pyspec/eth2spec/test/deneb/block_processing/test_process_execution_payload.py [hunk 1]
```diff
@@ -110,6 +151,22 @@ def test_incorrect_transaction_length_32_bytes(spec, state):
     yield from run_execution_payload_processing(spec, state, execution_payload, blob_kzg_commitments)
 
 
+@with_deneb_and_later
+@spec_state_test
+def test_no_transactions_with_commitments(spec, state):
```
Comment (hwwhww) [inline tests/core/pyspec/eth2spec/test/deneb/block_processing/test_process_execution_payload.py:156]: a good test case! 👍

**Review by hwwhww — APPROVED** [review]
lgtm

@marioevz thank you! 🚀

**Review by djrtwo — APPROVED** [review]
Thanks for jumping in @marioevz!

Things look good. @hwwhww will you add to the running spreadsheet?

**Comment by hwwhww** [comment]
@djrtwo done!

**Pull request merged**